spacialgaze-plugins/survey.js

```diff
@@ -144,8 +144,16 @@ class Survey {
 	}
 }
 
-function validateAnswer(room, message) {
+function validateAnswer(room, message, connection, user) {
+	let answer = this.repliers[i];
 	if (!room) return true;
+	if (Config.chatfilter) {
+		let filterResult = Config.chatfilter.call(this, answer, user, null, connection);
+		if (!filterResult) return;
+		if (answer !== filterResult) {
+			return this.errorReply("Your answer contains a globally banned word.");
```

Comment:
I dont think `this` refers to the command context here.

Reply:
My bad I did this all in a rush I did test the formatting portion not this validation for globally banned words